interfaces/xenmgr_host.xml

```diff
@@ -131,6 +131,9 @@
     <method name="shutdown">
       <tp:docstring>Shutdown the host device.</tp:docstring>
     </method>
+    <method name="shutdown_idle">
+      <tp:docstring>Called when idle period is determined to have passed</tp:docstring>
```

Comment:
I've only peeked at the xcpmd changes, and this is "Called when idle period is determined to have passed".  That reads to me like this method triggers the shutdown_idle action.  However it's use in xcpmd looks more like "returns a boolean stating whether or not Shutdown-on-Idle is enabled."  I haven't seen the xenmgr piece, but I think the docstring may need clarification.

Reply:
I think the naming could definitely be better, but in this case ```shutdown_idle``` is the RPC function call.  ```shutdown_idle``` is _also_ a xcpmd variable that lives in the db, defined in minutes, of how long the system must be idle before calling this function.  ```shutdown_on_idle``` is the boolean yes/no that tells xcpmd whether to care about the idle timeout.

Reply:
Now that I've looked more at the xcpmd changes,  I see that shutdown_idle method is called.  So the docstring is:
`<tp:docstring>Shutdown the host device.  Called when idle period is determined to have passed. </tp:docstring>`

How is this different than a plain `shutdown`?

Reply:
The ```shutdown_idle``` adds a timeout that will forcefully poweroff the box after a set period.  The thinking is, no user input for the specified time should shut off the box.  First, we should try to do a graceful shutdown and let all the guest domains clean up, but if a VM gets stuck for some reason, the box won't completely turn off.  The extra timeout ensures that the box will be poweroff no matter what, after gracefully trying to shutdown.

Reply:
Thanks.  Makes sense to force shutdown guests.

So the docstring should be something like:
`Shutdown the host devices.  VMs which fail to gracefully shutdown are forced off.`?

Reply:
Yes I can update the docstring to reflect that.